utils/cpuload/netlink.go

```diff
@@ -0,0 +1,232 @@
+// Copyright 2015 Google Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package cpuload
+
+import (
+	"bytes"
+	"encoding/binary"
+	"fmt"
+	"syscall"
+)
+
+const (
+	GENL_ID_CTRL            = syscall.NLMSG_MIN_TYPE
+	GENL_VERSION_CTRL       = 0x1
+	TASKSTATS_GENL_NAME     = "TASKSTATS"
+	CGROUPSTATS_CMD_ATTR_FD = 0x1
+	CTRL_ATTR_FAMILY_ID     = 0x1
+	CTRL_ATTR_FAMILY_NAME   = 0x2
+	CTRL_CMD_GETFAMILY      = 0x3
+)
+
+type GenMsghdr struct {
```

Comment:
I'm guessing all these structs are package-private? Should be lowercase if that's the case.